lib/govuk_request_id.rb

```diff
@@ -0,0 +1,13 @@
+class GovukRequestId
+  def self.insert(body)
+    return body unless body.present?
+
+    doc = Nokogiri::HTML::DocumentFragment.parse(body)
```

Comment:
Maybe we'd be better adding the ID within an HTML comment which would mean we could avoid Nokogiri and any potential fragility that might introduce. We could treat body just as a string then and append it to the end.